imgops/README.md

```diff
@@ -2,6 +2,9 @@
 Local version of the imgops service
 
 ## Requirements
+* [Nginx with with image filter module](http://nginx.org/en/docs/http/ngx_http_image_filter_module.html)
+  * Linux: `sudo apt-get install nginx`
```

Comment:
You'd also need to install the `nginx-extras` package to get the image filter module on linux.